stable/grafana/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 name: grafana
-version: 1.14.6
+version: 1.14.7
```

Comment:
new feature - 1.15.0

Reply:
also see @stealthybox comments

Reply:
this will need to be bumped higher -- I thin 1.17.0 or 1.18.0